5.5/examples/replica/persistent_nfs_volume.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
@bparees that boundary is very thin here as this is intended for users to try the replication template. That template won't work until the users (or admins) create the volume....